alignak_module_backend/broker/module.py

```diff
@@ -861,7 +864,7 @@ def manage_brok(self, brok):
                 logger.debug("Got contact update status brok: %s", brok.data)
                 self.update_status(brok)
 
-            if brok.type == 'new_conf':
+            if brok.type == 'new_conf' or not self.loaded:
```

Comment:
Ok. So any received brok will validate the `if` and then call the `get_refs` function....